ports/mimxrt/mpconfigport.h

```diff
@@ -57,7 +57,9 @@ uint32_t trng_random_u32(void);
 #define MICROPY_STACK_CHECK_MARGIN          (1024)
 #define MICROPY_ENABLE_EMERGENCY_EXCEPTION_BUF  (1)
 #define MICROPY_LONGINT_IMPL                (MICROPY_LONGINT_IMPL_MPZ)
+#define MICROPY_SCHEDULER                   (1)
```

Comment:
This is not a config option, so can be removed.

Reply:
Gone.